SQL/0000-00-00-schema.sql

```diff
@@ -1564,7 +1564,7 @@ CREATE TABLE `issues_history` (
   `issueHistoryID` int(11) unsigned NOT NULL AUTO_INCREMENT,
   `newValue` longtext NOT NULL,
   `dateAdded` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  `fieldChanged` enum('assignee','status','comment','sessionID','centerID','title','category','module','lastUpdatedBy','priority','candID', 'description','watching','instrument') NOT NULL DEFAULT 'comment',
```

Comment:
I don't think this is right. The CandID is the field that should be exposed to the user. The sequential CandidateID should *not* be leaked to the frontend anywhere, but only used for joins.

Reply:
After investigating, I think this is actually fine. As long as it doesn't get brought to the front-end (which it does), but it isn't because of its presence in this enum.